serverless.yml

```diff
@@ -1,28 +1,23 @@
-service: node-in-aws-web
-
-frameworkVersion: '2'
-
+service: my-first-app-rs-app
 provider:
   name: aws
   runtime: nodejs12.x
-  # setup profile for AWS CLI.
-  # profile: node-aws
+  lambdaHashingVersion: 20201221
+
 
 plugins:
   - serverless-finch
-  # 'serverless-single-page-app-plugin' is a custom plugin that located .serverless_plugins folder.
-  # Existing plugin (https://www.npmjs.com/package/serverless-single-page-app-plugin) doesn't have invalidate cache feature that often used during CI/CD events.
-  # How to build your own plugins: https://www.serverless.com/framework/docs/providers/aws/guide/plugins#service-local-plugin
   - serverless-single-page-app-plugin
 
 custom:
   client:
-    bucketName: node-in-aws-web-bucket
+    bucketName: front-part
```

Comment:
Pay attention to 'region'
Without specifying it it takes 'us-east-1' by default and you might end up deploying to wrong regions.